workspaces/jupyterlab/install.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+set -ex
```

Comment:
Forever wishing that Bash did this by default...

Reply:
You can add the `-ex` to the shebang line (`#! /bin/bash -ex`). From https://www.gnu.org/savannah-checkouts/gnu/bash/manual/bash.html#Invoking-Bash:

> All of the single-character options used with the `set` builtin (see [The Set Builtin](https://www.gnu.org/savannah-checkouts/gnu/bash/manual/bash.html#The-Set-Builtin)) can be used as options when the shell is invoked.